src/util.js

```diff
@@ -23,7 +23,7 @@ const getRound = (round) => {
 }
 
 const confirmUpdate = (id, round) => {
-  const feedback = takeTurn(id, round);
+  const feedback = takeTurn(id, round).result;
```

Comment:
I couldn't find a better way to return both the round and the feedback, so I ended up having to slightly alter the util.js file so that it would grab only the result for the feedback message.

Reply:
Found a better way and put it back! 😂